share/man/man4/lc.4
.\"	$OpenBSD: lc.4,v 1.12 2011/12/03 23:01:21 schwarze Exp $
.\"	$NetBSD: lc.4,v 1.4 1999/12/15 22:07:32 abs Exp $
.\"
.\" Copyright (c) 1997 The NetBSD Foundation, Inc.
.\" All rights reserved.
.\"
.\" This code is derived from software contributed to The NetBSD Foundation
.\" by Jason R. Thorpe of the Numerical Aerospace Simulation Facility,
.\" NASA Ames Research Center.
.\"
.\" Redistribution and use in source and binary forms, with or without
.\" modification, are permitted provided that the following conditions
.\" are met:
.\" 1. Redistributions of source code must retain the above copyright
.\"    notice, this list of conditions and the following disclaimer.
.\" 2. Redistributions in binary form must reproduce the above copyright
.\"    notice, this list of conditions and the following disclaimer in the
.\"    documentation and/or other materials provided with the distribution.
.\"
.\" THIS SOFTWARE IS PROVIDED BY THE NETBSD FOUNDATION, INC. AND CONTRIBUTORS
.\" ``AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED
.\" TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
.\" PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE FOUNDATION OR CONTRIBUTORS
.\" BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
.\" CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
.\" SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
.\" INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
.\" CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
.\" ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
.\" POSSIBILITY OF SUCH DAMAGE.
.\"
.Dd $Mdocdate: December 3 2011 $
.Dt LC 4
.Os
.Sh NAME
.Nm lc
.Nd DEC EtherWORKS III 10/100 Ethernet device
.Sh SYNOPSIS
.Cd "lc0 at isa? port 0x200"
.Cd "lc1 at isa? port 0x280"
.Sh DESCRIPTION
The
.Nm
device driver supports DEC EtherWORKS III Ethernet interfaces, which
are based on the LEMAC Ethernet chip.
This includes the DE203, DE204, and DE205.
.Sh MEDIA SELECTION
The EtherWORKS III series supports various combinations of media, depending
on model.
Some models also support media autoselect.
Media is selected with
.Xr ifconfig 8 Ns 's
.Sq media
directive.
.Sh SEE ALSO
.Xr arp 4 ,
.Xr ifmedia 4 ,
.Xr intro 4 ,
.Xr isa 4 ,
.Xr netintro 4 ,
.Xr hostname.if 5 ,
.Xr ifconfig 8
.Sh HISTORY
The
.Nm
driver first appeared in
.Nx 1.3 ,
.Ox
support was added in
.Ox 3.0 .
.Sh AUTHORS
.An -nosplit
The
.Nm
driver was written by
.An Matt Thomas Aq matt@netbsd.org
and ported to
.Ox
by
.An Niklas Hallqvist Aq niklas@openbsd.org
.Sh BUGS
The
.Nm
does not currently support EISA LEMAC interfaces.


gnu/usr.bin/perl/lib/unicore/lib/Gc/LC.pl
# !!!!!!!   DO NOT EDIT THIS FILE   !!!!!!!
# This file is machine-generated by lib/unicore/mktables from the Unicode
# database, Version 5.2.0.  Any changes made here will be lost!

# !!!!!!!   INTERNAL PERL USE ONLY   !!!!!!!
# This file is for internal use by the Perl program only.  The format and even
# the name or existence of this file are subject to change without notice.
# Don't use it directly.

# 'X' below means that this form is discouraged.  Consult perluniprops.pod
# 
# This file returns the 3207 code points in Unicode Version 5.2.0 that match
# any of the following regular expression constructs:
# 
#         \p{General_Category=Cased_Letter}
#         \p{Gc=LC}
# X       \p{Category=L_}
# X       \p{Is_General_Category=L&}
#         \p{Is_Gc=Cased_Letter}
#         \p{Is_Category=LC}
# 
#         \p{Cased_Letter}
#         \p{Is_Cased_Letter}
#         \p{LC}
#         \p{Is_LC}
#         \p{L_}
#         \p{Is_L_}
#         \p{L&}
#         \p{Is_L&}
# 
#     Meaning: [\p{Ll}\p{Lu}\p{Lt}]
# 
# perluniprops.pod should be consulted for the syntax rules for any of these,
# including if adding or subtracting white space, underscore, and hyphen
# characters matters or doesn't matter, and other permissible syntactic
# variants.  Upper/lower case distinctions never matter.
# 
# A colon can be substituted for the equals sign, and anything to the left of
# the equals (or colon) can be combined with anything to the right.  Thus,
# for example,
#         \p{Is_Category: Cased_Letter}
# is also valid.
# 
# The format of the lines of this file is: START\tSTOP\twhere START is the
# starting code point of the range, in hex; STOP is the ending point, or if
# omitted, the range has just one code point.  Numbers in comments in
# [brackets] indicate how many code points are in the range.

return <<'END';
0041	005A	 # [26]
0061	007A	 # [26]
00AA		
00B5		
00BA		
00C0	00D6	 # [23]
00D8	00F6	 # [31]
00F8	01BA	 # [195]
01BC	01BF	 # [4]
01C4	0293	 # [208]
0295	02AF	 # [27]
0370	0373	 # [4]
0376	0377	 # [2]
037B	037D	 # [3]
0386		
0388	038A	 # [3]
038C		
038E	03A1	 # [20]
03A3	03F5	 # [83]
03F7	0481	 # [139]
048A	0525	 # [156]
0531	0556	 # [38]
0561	0587	 # [39]
10A0	10C5	 # [38]
1D00	1D2B	 # [44]
1D62	1D77	 # [22]
1D79	1D9A	 # [34]
1E00	1F15	 # [278]
1F18	1F1D	 # [6]
1F20	1F45	 # [38]
1F48	1F4D	 # [6]
1F50	1F57	 # [8]
1F59		
1F5B		
1F5D		
1F5F	1F7D	 # [31]
1F80	1FB4	 # [53]
1FB6	1FBC	 # [7]
1FBE		
1FC2	1FC4	 # [3]
1FC6	1FCC	 # [7]
1FD0	1FD3	 # [4]
1FD6	1FDB	 # [6]
1FE0	1FEC	 # [13]
1FF2	1FF4	 # [3]
1FF6	1FFC	 # [7]
2102		
2107		
210A	2113	 # [10]
2115		
2119	211D	 # [5]
2124		
2126		
2128		
212A	212D	 # [4]
212F	2134	 # [6]
2139		
213C	213F	 # [4]
2145	2149	 # [5]
214E		
2183	2184	 # [2]
2C00	2C2E	 # [47]
2C30	2C5E	 # [47]
2C60	2C7C	 # [29]
2C7E	2CE4	 # [103]
2CEB	2CEE	 # [4]
2D00	2D25	 # [38]
A640	A65F	 # [32]
A662	A66D	 # [12]
A680	A697	 # [24]
A722	A76F	 # [78]
A771	A787	 # [23]
A78B	A78C	 # [2]
FB00	FB06	 # [7]
FB13	FB17	 # [5]
FF21	FF3A	 # [26]
FF41	FF5A	 # [26]
10400	1044F	 # [80]
1D400	1D454	 # [85]
1D456	1D49C	 # [71]
1D49E	1D49F	 # [2]
1D4A2		
1D4A5	1D4A6	 # [2]
1D4A9	1D4AC	 # [4]
1D4AE	1D4B9	 # [12]
1D4BB		
1D4BD	1D4C3	 # [7]
1D4C5	1D505	 # [65]
1D507	1D50A	 # [4]
1D50D	1D514	 # [8]
1D516	1D51C	 # [7]
1D51E	1D539	 # [28]
1D53B	1D53E	 # [4]
1D540	1D544	 # [5]
1D546		
1D54A	1D550	 # [7]
1D552	1D6A5	 # [340]
1D6A8	1D6C0	 # [25]
1D6C2	1D6DA	 # [25]
1D6DC	1D6FA	 # [31]
1D6FC	1D714	 # [25]
1D716	1D734	 # [31]
1D736	1D74E	 # [25]
1D750	1D76E	 # [31]
1D770	1D788	 # [25]
1D78A	1D7A8	 # [31]
1D7AA	1D7C2	 # [25]
1D7C4	1D7CB	 # [8]
END
